app/questionnaire/placeholder_parser.py

```diff
@@ -41,7 +41,7 @@ def __call__(self, placeholder_list: Sequence[Mapping]) -> Mapping:
         return self._placeholder_map
 
     def _lookup_answer(self, answer_id: str, list_item_id: str = None):
-        return self._answer_store.get_escaped_answer(answer_id, list_item_id)
+        return self._answer_store.get_escaped_answer_value(answer_id, list_item_id)
```

Comment:
If the `list_item_id` defaults to `None`, the `_lookup_answer` method is redundant.

Reply:
Removed the whole `_lookup_answer` method, thanks!